### ISSUE TITLE
UI design course: students' assignment

### ISSUE THREAD

**Issue by Arthur-Milchior** [ankidroid/Anki-Android#11664]
TL/DR we got a lot of feedback in there documents ([here](https://t.co/8Hu4NFSpJW), [here](https://t.co/UjMJPHAUqT), [here](https://t.co/D1fqL7bhwm) and [here](https://t.co/gWff8MtK7A)). Potentially more to come later. Some of those feedback may lead to improvement. So a first task is to split this feedback into actionable pieces in different github issues. Even if you disagree with the suggestion or feedback, please create an issue mentioning the suggested improvement, so that we can take the time to discuss it, and either use it or have a record of why we disagreed. Also mention #11664 in those issues, so that from this main issue, we can track all sub-issues

Long version: Educator Paul Hibbitts from Canadian's Simon Fraser University has been teaching this semestelr [User Interface Design](https://canvas.sfu.ca/courses/69678), and gave his student an [assignment](https://canvas.sfu.ca/courses/69678/assignments/751347) to conduct [a heuristic evaluation](https://balsamiq.com/learn/articles/usability-inspection/) and [thematic analysis](https://www.toptal.com/designers/ux-research/thematic-analysis-for-ux) to AnkiDroid .

I heard about it thanks to a [tweet](https://twitter.com/hibbittsdesign/status/1529930456279310336) with our name inside, and started a private message discussion from AnkiDroid's [twitter account](https://twitter.com/AnkiDroid) with M. Hibbitts. He agreed to share the documents with us, with the permission of his students, and authorization to use it if we see fit (and I explicitly stated to him that our open-collective has a fixed rate, whatever the kind of work, so we can potentially retribute if any is useful and used)

I told the educator that students can post their work themselves on github, or contact us on discord or by email. If they were to decide to do so later, they are free to claim their work as theirs (which may have the positive side of allowing us to ask more questions if, during developments, we realize something is unclear. And just to thanks them directly), otherwise we can use the documents as is. 

Personally, I've been using AnkiDroid from quite some time. While some issues were already known and currently adressed (e.g. browser is too minimal), there are other I have never thought abouts (the "+" becoming a "x" in the FAB is not intuitive), so in any case, I find cool to see detailed analysis from a fresh eye and with an expertise allowing to describe the issues in details.

**Comment by BrayanDSO** [comment]
I can't open any of the documents because I'm not on the `1sfu-my.sharepoint.com` directory

**Comment by Arthur-Milchior** [comment]
> I can't open any of the documents because I'm not on the `1sfu-my.sharepoint.com` directory

I changed links. Please reload the github issue and try the new ones

**Comment by Arthur-Milchior** [comment]
[third document](https://t.co/D1fqL7bhwm) received

**Comment by oyeraghib** [comment]
> To clarify, you'd like the documents split into 30 individual issues?

@bnknndy Yes, you can also check if the issue is already there or not. If not then creating a detailed issue on GitHub would be great so that discussion on specific issues could happen easily. And in future it would be nice to see **_"what were the issues reported"_** and **_"how were they resolved"_**. Please also join the [Discord](https://discord.com/invite/hpMsWaF) for more detailed discussions.

**Comment by bnknndy** [comment]
A quick note as I go through these: I see evidence that these researchers engaged the app deeply as users, to a degree that significantly outstrips the usual amount of time one might spend on a heuristic evaluation exercise even after accounting for a professional evaluator's experience level. This imbues their feedback with significant qualitative value.

Gathering this type of insight can be finnicky and time-consuming even for big companies. It's exceedingly rare in most other places, and it helps immensely when it comes to understanding an experience's users. If any of you happen to be reading this, thank you!

**Comment by Arthur-Milchior** [comment]
Fourth document in https://t.co/gWff8MtK7A

**Comment by Arthur-Milchior** [comment]
@bnknndy Can you please provide an update here. Did you looked at all files and created all issues? Is it still a work in progress you intend to resume? Are you done for now but works remains (totally acceptable), in this case, can you please explicitly state what remains to do so someone can take over?
